addons/web/static/src/search/search_bar/search_bar.js

```diff
@@ -181,6 +215,13 @@ export class SearchBar extends Component {
     async computeSubItems(searchItemId, query) {
         const searchItem = this.searchItems.find((i) => i.id === searchItemId);
         const field = this.fields[searchItem.fieldName];
+        const operator = searchItem.operator || "=";
+
+        if (field.type === "properties" && searchItem.type !== "field_property") {
+            await this._getPropertiesSearchItems(searchItem, field, query);
+            return [];
+        }
```

Comment:
It seems like the goal of this method is to return sub-items to push into "tasks".
It's a bit obscure but I'm guessing it's the sub-search results of the main filter items? Like the m2o suggestions for example?

Could we do it the same way for property fields and returns the items to push rather than hacking this method result and pushing items into "this.searchItems" directly ?

Seems a bit wild, and not really what you would expect from a method that is supposed to compute items.

Reply:
for properties, we need to generated searchItems and not subItems, so the code that do all the logic for normal field will be used for properties as well (so we have the exact same behavior)

I agree it shouldn't be in `computeSubItems`, so I move it in the caller function (but it's important to generate searchItems)